Flepper.Tests.Unit/QueryBuilder/Commands/WhereFilterTests.cs

```diff
@@ -339,5 +339,67 @@ public void ShoulContainLikeContainsAndStartsWithAndEndsWith()
             Assert.Equal("%abc2", parameters.@p1);
             Assert.Equal("abc3%", parameters.@p2);
         }
+
+        [Fact] //ShoulContainWhereWithNotEqualTo()
+        public void ShoulContainWhereWithBetween()
+        {
+            QueryResult result = FlepperQueryBuilder.Select()
+                .From("table")
+                .Where("field").Between(10, 20)
+                .BuildWithParameters();
+
+            result.Query.Trim()
+                .Contains("BETWEEN");
+            result.Query.Trim()
+                .Contains("@p0");
+            result.Query.Trim()
+                .Contains("@p1");
+            result.Query.Trim()
+                .Contains("[table]");
+            result.Query.Trim()
+                .Contains("[field]");
```

Comment:
Remove those lines, 351...360, the 365 is enough!

Reply:
Ok